[DOCS] Reformats multi search API (#46256)
elastic/elasticsearch: docs/reference/search/multi-search.asciidoc

[[search-multi-search]]
=== Multi Search API

> Allows to execute several search requests within the same API.

Reviewer: ```suggestion
Execute several search requests with a single API request.
```

[[search-multi-search-api-request]]
==== {api-request-title}

> `GET <index>/_msearch`

Reviewer: ```suggestion
`GET /<index>/_msearch`
```

[[search-multi-search-api-desc]]
==== {api-description-title}

> The multi search API allows to execute several search requests within the same 

Reviewer: ```suggestion
The multi search AP executes several searches from a single API request.
```

> API. The format of the request is similar to the bulk API format and makes use 

Reviewer: ```suggestion
The format of the request is similar to the bulk API format and makes use 
```

may be preceded by a carriage return `\r`. When sending requests to this endpoint the
`Content-Type` header should be set to `application/x-ndjson`.

[[search-multi-search-api-path-params]]
==== {api-path-parms-title}

include::{docdir}/rest-api/common-parms.asciidoc[tag=index]

[[search-multi-search-api-request-body]]
==== {api-request-body-title}

`aggregations`::
  (Optional, object) Specifies aggregations.
  

> `from`::

Reviewer: Believe this is in `common-parms.asciidoc`.

  (Optional, integer) Defines the starting offset. Defaults to `0`.
  
`max_concurrent_searches`::
  (Optional, integer) Specifies the maximum number of concurrent searches the 
  multi search API will execute. This default is based on the number of data 
  nodes and the default search thread pool size.
  
`max_concurrent_shard_requests`::
  (Optional, integer) Specifies the maximum number of concurrent shard requests 
  that each sub search request will execute per node. This parameter should be 
  used to protect a single request from overloading a cluster (for example a 
  default request will hit all indices in a cluster which could cause shard 
  request rejections if the number of shards per node is high). Defaults to `5`. 
  In certain scenarios parallelism isn't achieved through concurrent request 
  such that this protection will result in poor performance. For instance in an 
  environment where only a very low number of concurrent search requests are 
  expected it might help to increase this value to a higher number.

> `preference`::

Reviewer: Believe this is in `common-parms.asciidoc`.

  (Optional, string) Specifies the node or shard the operation should be 
  performed on. Random by default.
  
`query`::
  (Optional, <<query-dsl,query object>>) Defines the search definition using the 
  <<query-dsl,Query DSL>>.

> `routing`::

Reviewer: Believe this is in `common-parms.asciidoc`.
